packages/integration_test/lib/integration_test.dart

```diff
@@ -308,6 +308,8 @@ class IntegrationTestWidgetsFlutterBinding extends LiveTestWidgetsFlutterBinding
     // The engine could batch FrameTimings and send them only once per second.
     // Delay for a sufficient time so either old FrameTimings are flushed and not
     // interfering our measurements here, or new FrameTimings are all reported.
+    // TODO(CareF): remove this when flush FrameTiming is readly in engine.
+    //              See #64808
```

Comment:
A full URL to the issue might be better just in case the reader doesn't know the context of this number.

Reply:
Done.